example/src/main/deploy/docker/swarm_deploy.sh

```diff
@@ -36,7 +36,27 @@ echo ... done
 
 echo
 echo Deploying Artemis broker ...
-docker service create $CREATE_OPTIONS --name hono-artemis eclipsehono/hono-artemis:${project.version}
+docker secret create -l $NS broker.xml $CONFIG/hono-artemis-jar/etc/broker.xml
+docker secret create -l $NS bootstrap.xml $CONFIG/hono-artemis-jar/etc/bootstrap.xml
```

Comment:
maybe we could prefix these two secret names with `artemis-` as well in order to indicate their relation to Artemis?